form_data.js

```diff
@@ -1,30 +1,51 @@
-/* eslint-disable no-multi-str */
-/* eslint-disable no-plusplus */
-/* eslint-disable quotes */
-/* eslint-disable prefer-destructuring */
```

Comment:
Please use linter [config](https://github.com/microverseinc/linters-config/tree/master/html-css-js) only which is recommended for html+css+js.

Reply:
As you can see in my comment, I'm using the microverse's linters to check my code since the beginning. Thank you to approve my project, please!

Reply:
hey, you are wasting my request review for nothing the next will be the last one so take care of what you will do. You are not supposed to check which kind of linters I'm using (I believe that at this level everyone uses microverse linters), but  if the data in the form are saved in the browser local storage, check the requirement for this task [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md)